### ISSUE TITLE
Requirements on the Join-Leadership page is cut-off

### ISSUE THREAD

**Issue by 14r14** [TechOptimum/techoptimum#4]
The "Requirements" section for the web developer position is cut off (see screenshot). This could be an issue to do with responsive design. [Screenshot](https://drive.google.com/file/d/1bU4-Dl1qkHo0t49DvL7sFSNedVql2cAw/view?usp=sharing)

**Comment by rishaandesai** [comment]
> The "Requirements" section for the web developer position is cut off (see screenshot). This could be an issue to do with responsive design. [Screenshot](https://drive.google.com/file/d/1bU4-Dl1qkHo0t49DvL7sFSNedVql2cAw/view?usp=sharing)

Yeah, we're still adding content, so haven't gotten around to making most components responsive.

**Comment by siddharthd0** [comment]
wait, leave this open ig
